test/Microsoft.Health.Fhir.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -411,11 +407,11 @@ private async Task ExecuteAndValidate(Func<FhirClient, Task> clientSetup, HttpSt
             const string url = "Patient/123";
 
             // Create a new client with no token supplied.
-            var client = new FhirClient(_fixture.CreateHttpClient(), ResourceFormat.Json);
+            var client = (IVersionSpecificFhirClient)Activator.CreateInstance(_client.GetType(), _fixture.CreateHttpClient(), Format.Json, _fixture.FhirVersion);
```

Comment:
Maybe add a method to FhirClient (something like `SetupAnonymous` or something) to just reuse the same instance of the client but allow the token to be temporarily removed for a given call?